test/e2e/e2e_test.go

```diff
@@ -59,6 +59,8 @@ var _ = Describe("Manager", Ordered, func() {
 		Expect(err).NotTo(HaveOccurred(), "Failed to install CRDs")
 
 		By("deploying the controller-manager")
+		_ = os.Setenv("CLOUD_SDK_AK", "ak_test")
+		_ = os.Setenv("CLOUD_SDK_SK", "sk_test")
```

Comment:
>       Error from server (BadRequest): error when creating "STDIN": Secret in version "v1" cannot be handled as a Secret: illegal base64 data at input byte 2

https://github.com/HuaweiCloudDeveloper/cluster-api-provider-huawei/actions/runs/12683033377/job/35349451346?pr=26

Should this be base64 ?

Reply:
thank you for reminding